wolox-books/src/app/screens/book/components/list/list.component.html

```diff
@@ -1,11 +1,14 @@
 <div class="search">
   <input type="text" class="input-search" placeholder="Busc&aacute; por t&iacute;tulo del libro...">
 </div>
-<div *ngFor="let book of books" class="book-container">
+
+<div *ngFor="let book of books | async" class="book-container">
   <div class="book-card" *ngFor="let number of [0,1,2,3,4,5]">
     <img src="../../../../../assets/book-cover.png" alt="" class="book-cover">
     <p class="book-title">{{book.title}}</p>
     <p class="book-author">{{book.author}}</p>
-    <button type="button" class="add"><img src="../../../../../assets/add.png" alt=""></button>
+    <button type="button" class="add"><img src="../../../../../assets/add.png" alt="" (click)="addShoppingCart(book)"></button>
```

Comment:
close img tag

Reply:
ERROR in Errors parsing template: Void elements do not have end tags "img" ("tems">
    <img src="../../../../../assets/no-items.png" alt="no-items" class="empty-shopping-cart">[ERROR ->]</img>
    <div class="empty-text">
      <p class="empty-title">EL CARRITO EST&Aacute; VAC&Iacute;O<"): /Users/jorgediaz/Documents/projects/trainings/jd-angular/wolox-books/src/app/screens/shopping-cart/components/list/list.component.html@9:93